src/icons/Arrow/Arrow.tsx

```diff
@@ -0,0 +1,19 @@
+const Arrow = () => (
```

Comment:
Create a separate file for types `icon.typedefs.ts` with this content & add type to each `Icon`
```
import { DetailedHTMLProps, FC, SVGAttributes } from 'react';

export type SvgProps = DetailedHTMLProps<
  SVGAttributes<SVGElement>,
  SVGSVGElement
>;

export type FCIcon = FC<SvgProps>;

```